runtime/parachains/src/paras.rs

```diff
@@ -332,10 +335,58 @@ decl_error! {
 	}
 }
 
+decl_event! {
+	pub enum Event {
+		/// Current code has been updated for a Para. \[para_id\]
+		CurrentCodeUpdated(ParaId),
+		/// Current head has been updated for a Para. \[para_id\]
+		CurrentHeadUpdated(ParaId),
+		/// A code upgrade has been scheduled for a Para. \[para_id\]
+		CodeUpgradeScheduled(ParaId),
+		/// A new head has been noted for a Para. \[para_id\]
+		NewHeadNoted(ParaId),
+	}
+}
+
 decl_module! {
 	/// The parachains configuration module.
 	pub struct Module<T: Config> for enum Call where origin: <T as frame_system::Config>::Origin {
 		type Error = Error<T>;
+
+		fn deposit_event() = default;
+
+		#[weight = 0]
+		fn force_set_current_code(origin, para: ParaId, new_code: Vec<u8>) {
+			ensure_root(origin)?;
+			let validation_code = ValidationCode(new_code);
+			<Self as Store>::CurrentCode::insert(&para, validation_code);
+			Self::deposit_event(Event::CurrentCodeUpdated(para));
```

Comment:
@rphmeier is it important that i put the existing code into the `PastCode` storage?

What if there is data already there, I should overwrite it?